src/App.tsx

```diff
@@ -7,38 +7,108 @@ import classNames from 'classnames';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
+import { User } from './types/User';
+import { getUsers } from './api/users';
 import { Loader } from './components/Loader';
+import { getPosts } from './api/posts';
+import { Post } from './types/Post';
 
 export const App: React.FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
+  const [isProcessing, setIsProcessing] = useState(false);
+  const [isLoadingError, setIsLoadingError] = useState(false);
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [posts, setPosts] = useState<Post[] | null>(null);
+  const [isSidebarOpen, setIsSidebarOpen] = useState(false);
+  const [selectedPost, setSelectedPost] = useState<Post | null>(null);
+
+  const loadUsers = async () => {
+    try {
+      setIsLoadingError(false);
```

Comment:
you can remove this state update if it is already defined in useState as `false`, and has never been updated to `true` before by the moment this function called